### PR TITLE
fix build on PyPy 6.0

### PR DESCRIPTION
**Pull request by R4scal**
Hi.

This changes possible build on PyPy 6.0 (Python 3.5.3) AND Python 3.6.5

### CODE REVIEW

File: _scandir.c [hunk 1]
```diff
@@ -48,6 +48,11 @@ comment):
     PyUnicode_AsUnicode(unicode); *(addr_length) = PyUnicode_GetSize(unicode)
 #endif
 
+// Because on PyPy not worjing without
```
Comment (benhoyt) [inline _scandir.c]: Could you please fix the typo: worjing -> working

Comment (R4scal) [reply _scandir.c]: Thanks. Fix on b476572

**Comment by benhoyt** [comment]
Thanks! Want me to do a new release for this?

**Comment by R4scal** [comment]
As you want. For me it's not critical

**Pull request merged**